### PR TITLE
Add methods assigned to me for Iteration 3

### PR DESCRIPTION
**Pull request by javier-aguilar**
- [X] winningest_team
- [X] best_fans
- [X] worst_fans

Includes test files and truncated team csv file
Added helper methods (which are also tested)
Passes spec harness (under 5 seconds)

### CODE REVIEW

File: test/game_test.rb [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+  def test_it_checks_tie
+    assert_equal false, @game1.tie?
+    assert_equal false, @game2.tie?
+    assert_equal true, @game3.tie?
+  end
+
+end
+
+# === DONE ===
```
Comment (steve-meyers) [inline test/game_test.rb:81]: We still need to delete these comments

File: test/team_collection_test.rb [hunk 2]
```diff
@@ -1,9 +1,22 @@
...
 require "mocha/minitest"
 require "./lib/team_collection"
+require "./lib/game_collection"
 
 class TeamCollectionTest < Minitest::Test
 
+  def setup
+    @game_collection = GameCollection.new("./test/fixtures/games_truncated.csv")
+    @team_collection = TeamCollection.new("./test/fixtures/teams_truncated.csv")
+    # Chicago Team ID = 25
```
Comment (steve-meyers) [inline test/team_collection_test.rb:11]: Delete these comments?

**Review by steve-meyers — APPROVED** [review]
Everything looks good except a few leftover comments. It's also a little strange that this pull request is saying that a lot of the previously existing lines are new.

**Review by Shock253 — APPROVED** [review]
Looks good, although I don't know how I feel about the game collection and game classes being in this branch as well. I'd really like to get the game-collection merged into master so we can have this branch be consistient

**Comment by javier-aguilar** [comment]
Yeah, I must have accidentally git added them by mistake. Not sure how to un add those specific changes.

**Pull request merged**